thoth-tech/documentation
# Add Vale prose lint and CI integration

# Description

We want consistent use of tones and languages across our documentation. To automate the linting process and reduce manual effort in code review, we would like to introduce Vale linter to the workflow and integrate it with the CI pipeline. [Vale](https://docs.errata.ai/vale/about/) is a grammar, style, and word usage linter for the English language.

- Add convenient command lines for development
- Fix outstanding prose linting issues
- Add prose linting to CI
- Add Vale prose linting instructions on `CONTRIBUTING.md`

Relates to [Trello card](https://trello.com/c/GjagPSPF/66-implement-vale-prose-lint-for-markdown-documentation)

## Type of change


# How Has This Been Tested?

1. Checkout this branch
2. Follow the [instructions](https://github.com/thoth-tech/handbook/blob/feat%2Fvale-doc-lint/CONTRIBUTING.md#install-linters) to install the new `Vale` linter.
3. Run the following command
    ```shell
    npm run prose:check
    ```
4. We should expect to see something like the following in the terminal window
    ```shell
    > prose:check
    > vale --minAlertLevel error --output=docs/.vale/vale.tmpl docs

    0 errors, 0 warnings, and 0 suggestions found.
    ```

# Checklist:
